src/shared/components/modal/Modal.tsx

```
@@ -0,0 +1,86 @@
+          'bg-white rounded-[1.6rem]',
+        )}
+      >
+        <span className='title-sb-16 text-grayscale-900'>
+          {title}
+        </span>
+        <span className='mt-[0.7rem] caption-m-12 text-grayscale-700'>
+          {content}
+        </span>
+        <div className='flex gap-[1rem] mt-[1.6rem]'>
+          <Button
+            children={cancelChildren}
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)